gui/slick/views/home.mako

```diff
@@ -781,12 +776,13 @@ $(document).ready(function(){
             <span class="visible-print-inline">${download_stat}</span>
         </td>
 
-        <% show_size = sickbeard.helpers.get_size(curShow._location) %>
-        <td align="center" data-show-size="${show_size}">${sickbeard.helpers.pretty_filesize(show_size)}</td>
+        ## <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        ## <td align="center" data-show-size="${show_size}">${sickbeard.helpers.pretty_filesize(show_size)}</td>
+        <td align="center" data-show-size="0">Disabled!</td>
```

Comment:
Alright, still need to have the size check commented out but that should be good.